src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree.html

```diff
@@ -8,7 +8,11 @@ <h5>
                     {{tree.name}}
                 </a>
             </h5>
-            <button data-element="tree-item-options" class="umb-options btn-reset sr-only sr-only--focusable sr-only--hoverable" ng-hide="tree.root.isContainer || !tree.root.menuUrl" ng-click="options(tree.root, $event)" ng-swipe-right="options(tree.root, $event)">
+            <button data-element="tree-item-options"
+                    class="umb-options btn-reset sr-only sr-only--focusable sr-only--hoverable"
+                    aria-label="{{optionsText}} {{tree.name}}"
```

Comment:
Hello @emmaburstow - Lovely work on this issue 😃 

Instead of using the `aria-label` I'm thinking that perhaps the weird markup for the dots `<i></i><i></i><i></i>` could be refactored to `<span><span class="sr-only">{{optionsText}} {{tree.name}}</span>...</span>` and then some styling will probably be needed for the outer `<span>` to achieve the same visual effect for the ellipsis.

This way we can get rid of the less meaningful empty `<i>` tags 😃

Only a suggestion - The already applied work will solve the main issue just fine 👍

Reply:
Thanks Jan! I'll take a look at this tomorrow on the train home. Dinner first!

Reply:
Ace 🙌 Enjoy the dinner... I reckon it's the infamous lovely Yorkshire pudding tonight? 😋